avalon/maya/commands.py

```diff
@@ -18,8 +18,9 @@ def reset_frame_range():
     shot = io.find_one({"name": shot, "type": "asset"})
 
     try:
-        edit_in = shot["data"]["edit_in"]
-        edit_out = shot["data"]["edit_out"]
+
+        frame_start = shot["data"].get("frameStart", shot["data"]["edit_in"])
```

Comment:
This code is wrong. :)

Unlike an `or` statement I believe Python does not know how to optimize the out the second argument as such the full `shot["data"]["edit_in"]` is always retrieved. As susch, whenever only `frameStart` is in the data this code will still raise a KeyError:
```python
data = {"frameStart": 1}

print data.get("frameStart", data["edit_in"])
# KeyError

print data.get("frameStart", data.get("edit_in", None))
# Yay! :)
```